README.md

```diff
@@ -1,3 +1,15 @@
-# Northcoders News API
+# News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+The goal of this project is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
+
+To get your own local copy up and running, follow the steps below:
+
+1. Clone the repository:
+
+2. Navigate to the project directory.
+
+3. Install the neccessary dependencies.
+
+4. Create the '.env.development' and the '.env.test' files into your project and and add the correct database name into each file.
+
```

Comment:
This final instruction is unnecessary as .env.* in the .gitignore file is doing this job for us